contractsSol5/IKyberStorage.sol

```diff
@@ -7,16 +7,17 @@ import "./IKyberNetworkProxy.sol";
 
 /// @title KyberStorage interface
 contract IKyberStorage {
+
+    enum ReserveType {NONE, FPR, APR, BRIDGE, UTILITY, CUSTOM, ORDERBOOK, LAST}
+
     function setContracts(
         IKyberFeeHandler _feeHandler,
         address _matchingEngine
     ) external returns (bool);
 
     function setDAOContract(IKyberDAO _kyberDAO) external returns (bool);
 
-    function addReserve(address reserve, bytes32 reserveId)
-        external
-        returns (bool);
+    function addReserve(address reserve, bytes32 reserveId, ReserveType resType) external returns (bool);
```

Comment:
notice here code above is from pretify
So please use it on this file.

Reply:
udpated